### PR TITLE
fix: added translations of the word of 'Home' tab for each language

### LINKED ISSUE #346: i18n for home page
There is still English in `Home` page of the documentation even other language is selected.

image

Looks like strings are hard coded in the template of `home.html`. It would be better if home page can also be translated.

https://github.com/opengisch/QField-docs/blob/649911b210b11d3958bb4a66fa51b97ad0c9d3f7/theme_overrides/home.html#L37-L60

### PR DESCRIPTION
**Pull request by JinIgarashi**
@m-kuhn This PR is going to fix #346 partially. It is just for adding the word translation for `Home` tab name.  

I used translation software to add translations for `de`, `fr`, `it`, `pt`, `es` and `zh`. Could you double-check whether my translations are appropriate?

I tested every language locally. The following screenshot is for Japanese version.
image

### CODE REVIEW

**Comment by JinIgarashi** [comment]
@m-kuhn There is an error at `Install dependencies` section in Github Actions. Do you have any idea to fix this?

```shell
× git clone --filter=blob:none --quiet 'https://****@github.com/opengisch/mkdocs-material-insiders' /tmp/pip-install-anlq3h50/mkdocs-material_01d16c4deced4c639ca2849fb88af829 did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

Notice:  A new release of pip is available: 23.0.1 -> 23.1.2
Notice:  To update, run: pip install --upgrade pip
Error: Process completed with exit code 1.
```

**Comment by m-kuhn** [comment]
Thank you @JinIgarashi :heart: 

The error comes from secrets (access token) which are not available on pull requests.

@why-not-try-calmer is it ok to disable the zoho workflow on pull requests, it looked to me like we have an early exit in the script anyway?

https://github.com/opengisch/QField-docs/blob/649911b/utils/push_to_zoho.py#L187-L189

**Comment by why-not-try-calmer** [comment]
> 

It's absolutely ok to disable it for PRs!

File: mkdocs.yml [hunk 1]
```diff
@@ -201,36 +201,43 @@ plugins:
...
           Technical reference: Technische Referenz
           Success stories: Erfolgsgeschichten
         fr:
+          Home: Accueil
           Get started: Mise en route
           How-to guides: Tutoriels
           Technical reference: Référence technique
           Success stories: Témoignages de projets
         it:
+          Home: Casa
```
Comment (m-kuhn) [inline mkdocs.yml:216]: @domi4484 is this ok for "home page"?

Comment (domi4484) [reply mkdocs.yml:216]: No it sounds very strange :smile:, `Pagina iniziale` is more appropriated

Comment (JinIgarashi) [reply mkdocs.yml:216]: Thank you for checking my translations. It is not easy for me to choose an appropriate word even I use translation software.

Comment (JinIgarashi) [reply mkdocs.yml]: I fixed translation to `Pagina iniziale`

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**